tests/view_tests/tests/test_debug.py

```diff
@@ -248,12 +249,16 @@ def test_template_not_found_error(self):
 
 
 class ExceptionReporterTests(SimpleTestCase):
+    allow_database_queries = True
```

Comment:
We don't want to allow database queries in `SimpleTestCase` as the user will leak outside of the test case. I think creating a mock user like this should suffice:
```
class User(object):
    def __str__(self):
        return 'jacob'

request.user = User()
```